docs/website.md

```diff
@@ -0,0 +1,85 @@
+# Marble Website
+
+This will create the infrastructure to host the [front end website](https://github.com/ndlib/marble-website) for the project.
+
+This component uses the [unified-static-host.yml](/deploy/cloudformation/unified-static-host.yml) template. Below is the list of parameters that can be overridden in this template. Parameters with no default are required.
+
+| Parameter | Description | Default |
+|-----------|-------------|---------|
+| InfrastructureStackName | The name of the parent infrastructure/networking stack that you created. Necessary to locate and reference resources created by that stack. | marble-app-infrastructure |
+| DomainStackName | The name of the parent domain stack that you created. Necessary to locate and reference resources created by that stack. | marble-domain |
+| CreateDNSRecord | If True, will attempt to create a Route 53 DNS record for the CloudFront. | True |
+| HostnamePrefix | Hostname prefix for the website CDN |  |
+| EnvType | The type of environment to create, dev or prod. Affects default TTL | dev |
+
+To deploy this service, you will need to package an edge lambda, then create 3 stacks: a test stack, production stack, and a continuous delivery pipeline stack.
+
+## Test the edge lambda
+1. Install node 8.10
+2. Install Yarn
+
+```console
+cd src/unifiedEdgeLambda
+yarn install
+yarn run jest
+```
+
+## Package Edge Lambda
+```console
+cd src/unifiedEdgeLambda
+yarn install --production
+cd ../..
+aws cloudformation package \
+  --template-file deploy/cloudformation/unified-static-host.yml \
+  --s3-bucket $DEPLOY_BUCKET \
+  --output-template-file unified-static-host-output.yml
+```
+
+## Test
+```console
+aws cloudformation deploy \
+  --region us-east-1 \
+  --stack-name marble-website-test \
+  --template-file unified-static-host-output.yml \
+  --capabilities CAPABILITY_IAM \
+  --parameter-overrides EnvType=prod HostnamePrefix='marble-test'
+```
+
+## Production
+```console
+aws cloudformation deploy \
+  --region us-east-1 \
+  --stack-name marble-website-prod \
+  --template-file unified-static-host-output.yml \
+  --capabilities CAPABILITY_IAM \
+  --parameter-overrides EnvType=prod HostnamePrefix='marble'
+```
+
+## Continuous Delivery Pipeline
+This will create a CodePipeline that will monitor Github for changes and deploy those changes to a test stack, then to a production stack, so it expects the two different stacks above to exist.
+
+Before you begin see https://developer.github.com/v3/auth/#via-oauth-tokens for how to generate an OAuth token for use with these pipelines.
```

Comment:
What scopes will be required for the new token that I create?

Reply:
Added a note to all of the pipeline docs. Since we're targeting our public repos by default, you only need to add `public_repo` scope. If someone points it at a private repo they'll have to add full `repo` scope.